### ISSUE TITLE
Serious bug in __touches__ behavior

### ISSUE THREAD

**Issue by finetuned89** [sdispater/orator#155]
Say I have a model like this:

```
class Child(Model):
    __touches__ = ['parent']

    @belongs_to
    def parent(self):
        return Parent
```

If the `Parent` relation is **optional**, and a `Child` without an associated `Parent` is updated, **all the `Parent` items' timestamps are updated**. This is a serious problem in my view and should be resolved. If there is no `Parent` associated with the `Child`, none of the `Parent` items should have their timestamps updated.

**Comment by sdispater** [comment]
Thanks for reporting this.

I will look into it and get back to you.

**Comment by sdispater** [comment]
It should be fixed on the `0.9` branch. It will land in the next `0.9.6` release.
